vcell-core/src/test/java/org/vcell/sbml/TLogger.java

```diff
@@ -31,6 +32,7 @@ public void sendMessage(Priority p, ErrorType et, String message)
 			lowPriority.add(msg);
 		}
 		System.err.println(p + " " + et + ": "  + message);
+		throw new VCLoggerException(message);
```

Comment:
again here the intent is to collect messages for later processing, but not to throw an exception and abort.